org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -174,11 +197,26 @@ public SpaceMode DistanceSpaceMode
         /// </summary>
         private float WorldToLocalScale => transform.InverseTransformVector(transform.forward).magnitude;
 
+        /// <summary>
+        /// Holds the duples Collider + Interactor that have triggered proximity.
+        /// </summary>
+        private HashSet<(Collider, XRBaseInteractor)> activeCollidersWithInteractor = new HashSet<(Collider, XRBaseInteractor)>();
+
         /// <summary>
         /// If the <see cref="GetSelectionProgress"/> value is smoothed to within this threshold of 0 or 1, the <see cref="GetSelectionProgress"/> will snap to 0 or 1.
         /// </summary>
         private const float selectionProgressEpsilon = 0.00001f;
 
+        /// <summary>
+        /// Indicates whether anything is hovering on the button (true) or not (false).
+        /// </summary>
+        private bool isHovered = false;
```

Comment:
This is hiding the base isHovered variable.

You should be able to just use the base IsHovered value.

Reply:
I missed that, thanks for catching it and letting me know.  I removed it in the latest commit.  Resolving this.